imports/ui/pages/questions_library.jsx

```diff
@@ -389,69 +391,49 @@ export const QuestionsLibrary = createContainer(props => {
   const handle =  Meteor.subscribe(subscription, props.courseId)
   const courseId = props.courseId
   
```

Comment:
Don't call the prop "library", since that makes it sound like it goes to question.library, maybe "libraryType"?